app/models/search_query.rb

```diff
@@ -68,7 +71,14 @@ def update_for_click(position)
     save!
   end
 
+  def self.record_list(user, path, params)
+    self.create!(user_id: user.id,
+                 list_path: path,
+                 list_params: params.to_yaml,
```

Comment:
Is there a reason we need YAML here and can't just use a hash (which I _think_ would just be converted to JSON)?

Reply:
@beccapearce has a feeling that this is required for something so 👍